client.go

```diff
@@ -39,6 +39,8 @@ type Client interface {
 
 	// Add logical port PORT on SWITCH
 	LSPAdd(ls string, lsp string) (*OvnCommand, error)
+	// Link logical port to router
+	LSPLinkToRouter(ls string, lsp, lrp string) (*OvnCommand, error)
```

Comment:
It is creating a new LSP and link to an existed LRP, so the API is better to be named as LSPAddToLRP.